### PR TITLE
Update link to devs_ux channel on discord

### PR DESCRIPTION
**Pull request by MatzeG1t**
the old link was an invitation, this link was created just now from the discord app (non invitation)

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by KTibow** [comment]
the link you sent is a link to a message that only works if the user is already in there. instead consider linking to https://www.home-assistant.io/join-chat/

File: gallery/src/pages/concepts/home.markdown [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Home Assistant interface is based on Material Design. It's a design system c
 
 We want to make it as easy for designers to contribute as it is for developers. There’s a lot a designer can contribute to:
 
-- Meet us at <a href="https://discord.gg/BPBc8rZ9" rel="noopener noreferrer" target="_blank">devs_ux Discord</a>. Feel free to share your designs, user test or strategic ideas.
+- Meet us at <a href="https://discord.com/channels/330944238910963714/867715211619074048" rel="noopener noreferrer" target="_blank">devs_ux Discord</a>. Feel free to share your designs, user test or strategic ideas.
```
Comment (frenck) [inline gallery/src/pages/concepts/home.markdown:21]:
```suggestion
- Meet us at <a href="https://www.home-assistant.io/join-chat" rel="noopener noreferrer" target="_blank">devs_ux Discord</a>. Feel free to share your designs, user test or strategic ideas.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @MatzeG1t 👍 

Please note, you need to sign our CLA (as the bot above mentioned) before we can merge this. Can you take a look?

../Frenck

**Review by bramkragten — APPROVED** [review]

**Comment by bramkragten** [comment]
Stale, updated the link

**Pull request closed without merging**